modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/bar/SimpleBarChart.es.js

```diff
@@ -32,9 +32,11 @@ describe('SimpleBarChart', () => {
 	afterEach(cleanup);
 
 	it('renders', () => {
-		const {queryByText} = render(<SimpleBarChart {...props} />);
+		const {queryAllByText, queryByText} = render(
+			<SimpleBarChart {...props} />
+		);
 
-		expect(queryByText('Option 1')).toBeTruthy();
+		expect(queryAllByText('Option 1')).toBeTruthy();
```

Comment:
@alinedoleron, why did you just need to use `queryAllByText` here?

Reply:
Because after adding the customised axis tick, a new tag appeared. 

`<span id="recharts_measurement_span" style="position: absolute; top: -20000px; left: 0px; padding: 0px; margin: 0px; border: none; white-space: pre;">Option 1</span>`


Its content has the innerHtml equals to 'Option 1'. So the test failed because it founded two cases of 'Option 1', and it suggested that, if  finding more than one case of Option 1 were intentional, I should add queryAllByText instead of Option 1.

Reply:
@alinedoleron, can you please verify why it happened just for `Option 1`?